packages/syft/src/syft/service/api/api.py

```diff
@@ -568,16 +568,6 @@ def exec_code(
             # run all this code to clean up the state
             code.update_state(internal_context.state)
 
-            if isinstance(code, PublicAPIEndpoint):
-                self.mock_function = code
-            else:
-                self.private_function = code  # type: ignore
-
-            api_service = context.server.get_service("apiservice")
-            api_service.stash.upsert(
-                context.server.services.user.root_verify_key, self
-            ).unwrap()
-
```

Comment:
This code is no-op since we always call `exec_code` with either self.mock_function or self.private_function.